paparazzi-gradle-plugin/src/test/projects/verify-data-binding/src/main/java/androidx/databinding/DataBinderMapperImpl.kt

```diff
@@ -0,0 +1,17 @@
+package androidx.databinding
+
+import android.annotation.SuppressLint
+
+/**
+ * Running instrumented tests with Robolectric on Data Binding enabled library modules throws [NoClassDefFoundError] for
```

Comment:
Snagged this from an associated thread about Robolectric having issues finding the same DataBinderMapperImpl. I can redo the comment here and link to the original thread for more context

Reply:
Yeah that's a big concern. ByteBuddy could work as long as we can access the module's package to find the generated class

Reply:
So this is interesting.  Note that in https://github.com/cashapp/paparazzi/issues/1016, the repro project is an Android application, not a library project.  This is because DataBinding generates `DataBinderMapperImpl` in the app module.

Now we could support that, but coupled with my comment about `<data>`, to what cause?

I'm of the opinion that we should strive to support ViewBinding style inflations, via`FooBinding.inflate(..)`, but not the full suite of possibilities that DataBinding offers.  Thoughts?

Reply:
Nonetheless, if the feeling is a resounding "yes, we should support databinding", then I think the way to tackle this unpleasant dev experience would be to check for the Gradle option being set in the buildFeatures, pass that as a System Property ("paparazzi.androidx.databinding.enabled") and load this impl in Paparazzi.kt when that property is set.

Reply:
I think we still need the `DataBinderMapperImpl` to support ViewBinding so I can look into setting up a system property